client/src/pages/Login/Login.tsx

```diff
@@ -37,24 +36,23 @@ export default function Login(): JSX.Element {
 
   return (
     <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} elevation={6} component={Paper} square>
         <Box
           display="flex"
           justifyContent="space-between"
           alignItems="flex-start"
           flexDirection="column"
           className={classes.authWrapper}
         >
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
-              </Grid>
-            </Grid>
+          {/* <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" /> */}
+          <Box width="100%" maxWidth={450} p={3} alignSelf="center" elevation={6} component={Paper} square>
             <LoginForm handleSubmit={handleSubmit} />
+            <Box p={2} alignSelf="center" style={{ textAlign: 'center' }}>
+              <span>I do not have an account </span>
+              <Link to="/signup" style={{ display: 'inline', color: '#f14140' }}>
+                sign up
+              </Link>
+            </Box>
           </Box>
           <Box p={1} alignSelf="center" />
```

Comment:
Check whether this is still necessary, might've only been useful for the old UI.